rules/prefer-starts-ends-with.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+const doesNotContain = (string, chars) => chars.every(char => !string.includes(char));
+
+const isSimpleString = string => doesNotContain(
+	string,
+	['^', '$', '+', '[', '{', '(', '\\', '.', '?', '*']
```

Comment:
Unless someone happens to find a handy regex parser, I think we should drop support for regexes until the time we find one or someone happens to find a different solution.

By the way @firasdib, I would love to see the one used in regex101 as a library :D